.travis.yml

```diff
@@ -15,6 +14,20 @@ notifications:
   email:
     - yic@twitter.com
```

Comment:
non blocking. feel free to add your email if you wish.

Reply:
Absolutely, I wasn't sure what the protocol was for that.